pkg/install/install.go

```diff
@@ -111,6 +122,14 @@ func (i Installer) Install(d *deployment.Deployment) (err error) {
 		return err
 	}
 
+	err = i.b.Install(trans.Path, d)
```

Comment:
I think this should be replaced after the config script hook, this way if some additional config (e.g. a new initrd) is applied the bootloader installer can actually pick the updated data.

Reply:
Interesting, I put this before the config-script to be able to make changes to the bootloader setup, but I don't have strong feelings about this so I will move it to after the hook!